### ISSUE TITLE
Required numpy version incompatible with current pandas

### ISSUE THREAD

**Issue by coventry** [probcomp/bdbcontrib#108]
The requirements in `setup.py` should specify a fixed pandas version compatible with numpy version, which is currently fixed at 1.8.2, or the precipitating upstream bug in numpy should be fixed or worked around so numpy can be brought up to date.  The current arrangement results in the following error during `pip` installation of `bdbcontrib` when it tries to import pandas:

```
ubuntu@ip-172-31-17-62:~$ python -c 'import pandas'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/pandas/__init__.py", line 7, in <module>
    from pandas import hashtable, tslib, lib
  File "pandas/src/numpy.pxd", line 157, in init pandas.hashtable (pandas/hashtable.c:38276)
ValueError: numpy.dtype has the wrong size, try recompiling
```
I believe this is the problem because `import pandas` works after `pip install pandas`, then fails during and after `pip install bdbcontrib`.

**Comment by coventry** [comment]
BTW, the precipitating bug is [corrected in seaborn master](https://github.com/mwaskom/seaborn/issues/735).

**Comment by coventry** [comment]
The tests pass [mostly](https://github.com/probcomp/bdbcontrib/issues/109) after

```
pip install git+https://github.com/mwaskom/seaborn
```

**Comment by gregory-marton** [comment]
Sorry to reply so late.

This isn't because the versions are incompatible, but because you got
pandas via a pre-compiled binary (wheel) rather than recompiling with the
older numpy. Try installing pandas with --no-cache-dir.

Grem


On Wed, Jan 20, 2016 at 11:27 PM, coventry <notifications@github.com> wrote:

> The requirements in setup.py should specify a fixed pandas version
> compatible with numpy version, which is currently fixed at 1.8.2, or the
> precipitating upstream bug in numpy should be fixed or worked around so
> numpy can be brought up to date. The current arrangement results in the
> following error during pip installation of bdbcontrib when it tries to
> import pandas:
>
> ubuntu@ip-172-31-17-62:~$ python -c 'import pandas'
> Traceback (most recent call last):
>   File "<string>", line 1, in <module>
>   File "/usr/local/lib/python2.7/dist-packages/pandas/__init__.py", line 7, in <module>
>     from pandas import hashtable, tslib, lib
>   File "pandas/src/numpy.pxd", line 157, in init pandas.hashtable (pandas/hashtable.c:38276)
> ValueError: numpy.dtype has the wrong size, try recompiling
>
> I believe this is the problem because import pandas works after pip
> install pandas, then fails during and after pip install bdbcontrib.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/probcomp/bdbcontrib/issues/108>.
>



-- 
Gregory Marton
--
gremio@mit.edu
617-858-0775
http://csail.mit.edu/~gremio/cal
